src/normalizeManifest.ts

```diff
@@ -6,7 +6,7 @@ export interface Manifest {
   master_signature: string
   seq: number
   domain?: string
-  ephemeral_key?: string
+  signing_key?: string
```

Comment:
Why'd we change to `signing_key`?

Reply:
I changed it in order to remain consistent with what's in the explorer and the database from validations-etl. Also I think using ephemeral_key introduces a new manifest format while using signing_key keeps it mostly consistent with the manifests stream.